warp/tests/locality/example_fluid_locality.py

```
@@ -0,0 +1,326 @@
+        self.div = wp.zeros_managed(shape, dtype=float)
+
+        # capture pressure solve as a CUDA graph
+        # self.use_cuda_graph = wp.get_device().is_cuda
+        self.use_cuda_graph = False
+        if self.use_cuda_graph:
+            with wp.ScopedCapture() as capture:
+                self.pressure_iterations()
+            self.graph = capture.graph
+
+        # create a stream for each device
+        # self.streams = [wp.Stream(device=f"cuda:{i}") for i in range(wp.get_cuda_device_count())]
```

Ruff (0.14.2)

Comment contains ambiguous ` ` (NO-BREAK SPACE). Did you mean ` ` (SPACE)?

(RUF003)